src/flow_solver/flow_solver_cases/periodic_1D_unsteady.cpp

```diff
@@ -21,24 +11,74 @@ namespace FlowSolver {
 template <int dim, int nstate>
 Periodic1DUnsteady<dim, nstate>::Periodic1DUnsteady(const PHiLiP::Parameters::AllParameters *const parameters_input)
         : PeriodicCubeFlow<dim, nstate>(parameters_input)
+        , unsteady_data_table_filename_with_extension(this->all_param.flow_solver_param. unsteady_data_table_filename+ ".txt")
 {
+
+}
+
+
+template <int dim, int nstate>
+double Periodic1DUnsteady<dim, nstate>::compute_energy_collocated(
+        const std::shared_ptr <DGBase<dim, double>> dg
+        ) const
+{
+    // Intention is to eventually calculate energy in physics and generalize to arbitrary nodes
```

Comment:
Fine for now, you could take a look at `periodic_turbulence.cpp` to see how its done

Reply:
Yep, I have looked there and was planning to use that as a template! It just necessitates adding an energy integration function to `burgers.h`, which I saw as out of scope of this PR.

Reply:
I think the actual integration operation should be done in `src/flow_solver_cases/periodic_cube_flow.cpp` and somehow allow all derived classes to pass what should be getting integrated -- so that we minimize code repetition. Then `burger.h` simply defines the energy calculation at a single point.